tools/js/extractPcEntityMetadata.js

```diff
@@ -0,0 +1,153 @@
+const fs = require('fs')
```

Comment:
I'd rather we have a dedicated package for extraction. tools/js should not contain large amount of code as it's not documented and generally not maintained.

where would that kind of extraction fit? what's the source of data ?

Reply:
Yeah, I didn't know where else to put it so I left it in here for now.

It basically clones a copy of the mc source code and then does some string manipulation to extract the relevant information from the human readable source code files. Then updates the files in mc-data.

http://github.com/prismarinejs/mcdataextracting requires a fabric mode and Java code injection
http://github.com/prismarinejs/minecraft-jar-extractor is a better fit, but would need some refactoring to use mojang mappings and the [ForgeFlower](https://github.com/MinecraftForge/ForgeFlower) decompiler

Reply:
yeah I was not saying reimplement it another way, just move it elsewhere

maybe it could just be put in minecraft-jar-extractor with no change

Reply:
but let's not block on that, can be done in another pr